src/repository/ProductDatabaseRepository.js

```diff
@@ -20,9 +27,7 @@ class ProductDatabaseRepository extends BaseRepository
     await this._database.insertOne(this._collectionName, {
       ...(await this._formatProduct(model)),
       created_at: new Date(),
-      // save it as string to be able to search in getByIDOrSlug() method
```

Comment:
I decided to choose another way, since saving that into string creates inconsistency in responses, and format every response is oughhh...